README.md

```diff
@@ -75,34 +75,61 @@ The inventory service is an independent web service that accepts orders via HTTP
 
 ### Developing the Ballerina services with circuit breaker
 
-The `ballerina/http` package contains the circuit breaker implementation. After importing that package you can create a client with a circuit breaker. The `endpoint` keyword in Ballerina refers to a connection with a remote service. The following code segment is adding the circuit breaker capabilities for the endpoint.
+The `ballerina/http` package contains the circuit breaker implementation. After importing that package you can create
+a client with a circuit breaker. `http:Client` allows establishing a connection with a remote party (endpoint), circuit breaker configurations could be provided while initializing the client. 
+The following code segment will add the circuit breaker capabilities for the endpoint.
```

Comment:
```suggestion
The following code segment adds the circuit breaker capabilities for the endpoint.
```

Changing the tense from future tense to simple present. Normally the word "will" indicates that the tense is future. It's good if we can apply this everywhere.